### PR TITLE
optimise support

### PR DESCRIPTION
**Pull request by katzio**

### CODE REVIEW

File: faq/feedback-submitted.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,500" rel="stylesheet">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css" integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS"
+        crossorigin="anonymous">
+    <link rel="stylesheet" href="./style.css">
+
+    <title>Help request received</title>
```
Comment (berryve) [inline faq/feedback-submitted.html]: According to zeplin - there is no title here

Comment (katzio) [reply faq/feedback-submitted.html]: Thats only the title of the page successful form submission

File: faq/html-generator/contact_us.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import os
+from yattag import Doc, indent
+
+GEN_CODE_HTML = "<!-- GENERATED CODE -->"
+GEN_CODE_JS = "/** GENERATED CODE */"
+
+
+def generate_cotagory_menu(category_list):
```
Comment (berryve) [inline faq/html-generator/contact_us.py]: typo: it's category not cotagory

File: faq/html-generator/templates/contact-us.html [hunk 3]
```diff
@@ -2,81 +2,73 @@
 <html lang="en">
 
 <head>
+    <base href="https://s3.amazonaws.com/kinapp-static/faq2/" />
```
Comment (berryve) [inline faq/html-generator/templates/contact-us.html]: don't forget cdn.kinitapp.com

File: faq/assets/database.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (berryve) [inline faq/assets/database.json]: suggest to rename the file to contents.json

**Pull request merged**